tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -250,41 +163,4 @@ describe('Acceptance: addon-smoke-test', function() {
       handleError(error, 'npm');
     });
   });
-
-  function wait(time) {
-    return new Promise(function(resolve) {
-      setTimeout(function() {
-        resolve();
-      }, time);
-    });
-  }
-
-  var isWindows = /^win/.test(process.platform);
-
-  (isWindows ? it.skip : it)('doesn\'t fail to build new files', function() {
```

Comment:
What's the story about the bug that made us add this test? How likely is it to reoccur? (It looks like we're testing across the testem boundary here. Maybe @johanneswuerbach should be aware that we're not doing this anymore...)